src/platform/packages/shared/kbn-apm-synthtrace/src/scenarios/simple_otel_logs.ts

```diff
@@ -0,0 +1,246 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import {
+  OtelLogDocument,
+  generateLongId,
+  generateShortId,
+  otelLog,
+} from '@kbn/apm-synthtrace-client';
+import moment from 'moment';
+import { Scenario } from '../cli/scenario';
+import { withClient } from '../lib/utils/with_client';
+import {
+  MORE_THAN_1024_CHARS,
+  getCloudProvider,
+  getCloudRegion,
+  getCluster,
+  getIpAddress,
+  getServiceName,
+} from './helpers/logs_mock_data';
+
+// Logs Data logic
+const MESSAGE_LOG_LEVELS = [
+  { message: 'A simple log with something random <random> in the middle', level: 'info' },
+  { message: 'Yet another debug log', level: 'debug' },
+  { message: 'Error with certificate: "ca_trusted_fingerprint"', level: 'error' },
+];
+
+const scenario: Scenario<OtelLogDocument> = async (runOptions) => {
+  const constructLogsCommonData = () => {
+    const index = Math.floor(Math.random() * 3);
+    const serviceName = getServiceName(index);
+    const logMessage = MESSAGE_LOG_LEVELS[index];
+    const { clusterId, clusterName, namespace } = getCluster(index);
+    const cloudRegion = getCloudRegion(index);
+
+    const commonLongEntryFields: OtelLogDocument = {
+      trace_id: generateLongId(),
+      resource: {
+        attributes: {
+          'service.name': serviceName,
+          'service.version': '1.0.0',
+          'service.environment': 'production',
+          'k8s.cluster.name': clusterName,
+          'k8s.cluster.id': clusterId,
+          'k8s.namespace.name': namespace,
+          'k8s.pod.name': `${serviceName}-${generateShortId()}`,
+          'cloud.provider': getCloudProvider(),
+          'cloud.region': cloudRegion,
+          'cloud.availability_zone': `${cloudRegion}a`,
+          'cloud.account.id': generateShortId(),
+          'cloud.resource_id': generateShortId(),
+        },
+      },
+      attributes: {
+        'log.file.path': `/logs/${generateLongId()}/error.txt`,
+      },
+    };
```

Comment:
We had the same situation in APM. What we ended up doing was creating a pipeline that transforms ECS docs into what we expect from Otel docs: https://github.com/elastic/kibana/pull/217019

There is a [pipeline](https://github.com/elastic/kibana/blob/main/src/platform/packages/shared/kbn-apm-synthtrace/src/lib/apm/client/apm_synthtrace_es_client/otel/apm_to_otel_pipeline.ts#L35) that does that, and we can easily tell [synthtrace which pipeline to use](https://github.com/elastic/kibana/blob/main/src/platform/packages/shared/kbn-apm-synthtrace/src/scenarios/span_links.ts#L46). The advantage with this approach is that it allows us to reuse existing scenarios. Apparently, this could also be done for logs.

Reply:
Fair suggestion, I'm not sure whether that will make more sense or whether we want to diverge further for the otel cases in the future. Let's start simple with a fork so we don't have to think through all the conversion logic (I skimmed through the stuff you have there and we would need a couple other things for logs in particular), if it keeps coming up we can still generalize.

I hope that it will stay a one-off thing and streams / "slash logs" will be what most people use anyway 🙂